cmd/prometheus/main.go

```diff
@@ -439,8 +508,8 @@ func computeExternalURL(u, listenAddr string) (*url.URL, error) {
 		u = fmt.Sprintf("http://%s:%s/", hostname, port)
 	}
 
-	if startsOrEndsWithQuote(u) {
-		return nil, fmt.Errorf("URL must not begin or end with quotes")
+	if ok := govalidator.IsURL(u); !ok {
```

Comment:
What is the reason to revert #3199 and bring back govalidator? This seems to be outside of the scope of this PR.

Reply:
not sure what happened will fix it.